docs/il-curriculum/il-curriculum-vitae-per-un-developer

```diff
@@ -0,0 +1,17 @@
+Se stai leggendo questo capitolo probabilmente ti stai guardando attorno, alla ricerca di un nuovo lavoro, una nuova collaborazione, una nuova RAL (si spera più alta dell'attuale).
+Lo so, siamo nel 2023, davvero il CV si usa ancora? Ebbene sì! Il CV rimane lo strumento fondamentale (ma va ovviamente affiancato ad altri strumenti che vedremo in seguito)
+per proporsi a chi sta offrendo lavoro.
+Partiamo da un assunto fondamentale: il CV perfetto NON ESISTE, così come non esiste “il CV a prova di ATS” (Applicant Tracking System). 
+Le indicazioni che troverai in questo capitolo saranno squisitamente di tipo tecnico/qualitativo/estetico, per darti la possibilità
+di costruire un CV funzionale ed attrattivo, contenente il giusto mix di parole chiave, descrizione delle esperienze professionali, soft skill e molto altro. 
+
+Alcuni consigli utili prima di entrare nel vivo:
+
+- È bene cercare di costruire il CV di una sola pagina (un recruiter esperto ci mette circa 10–20 secondi per leggerne uno) per catturare subito 
+  l’interesse e l’attenzione di chi riceverà il documento. Content is king, ma anche l’occhio vuole la sua parte!
```

Comment:
```suggestion
  l’interesse e l’attenzione di chi riceverà il documento. _Content is king_, ma anche l’occhio vuole la sua parte!
```

Essendo un libro in italiano, le citazioni e gli inglesismi credo sia corretto siano ben identificati.